Detour/Source/DetourNavMeshQuery.cpp

```diff
@@ -721,8 +721,13 @@ dtStatus dtNavMeshQuery::findNearestPoly(const float* center, const float* exten
 	if (dtStatusFailed(queryPolygons(center, extents, filter, polys, &polyCount, MAX_SEARCH)))
 		return DT_FAILURE | DT_INVALID_PARAM;
 	
+	if (polyCount == 0)
+		return DT_SUCCESS;
```

Comment:
Oh whoops, sorry! I'm imagining PR #102 already being merged in :) Ok; the combination of this PR and #102 should ensure that `*nearestRef = 0` in this case.